README.rst

```diff
@@ -2,108 +2,260 @@
+``plone/staticresources/static/`` into minified versions with source maps and, for JavaScript only, also into an
+un-minified version. The output directory is configurable by using ``csscompilation`` and ``jscompilation`` attributes
+of the bundle registration.
 
-For Patternslib, visit: http://github.com/patternslib/Patterns
+Bundles are groups of resources. By default Plone comes configured to serve two main bundles:
 
+- ``plone``: containing JS and CSS used for anonymous visitors - ``plone-logged-in``: with additional resources for
+  authenticated visitors, e.g. for editor and management user interfaces
 
-How to upgrade the resources in this package
---------------------------------------------
+Finally Plone encapsulates most of its CSS and JS into units called Patterns that live in the ``mockup`` package.
+Although Patterns provide their own registered Resources, those are not registered to Plone main bundles. Instead, they
+are imported through native LESS and Require JS "entry points" that live in ``plone.staticresource``:
+
+- `plone/staticresources/static/plone.js
+  <https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone.js>`_
+- `plone/staticresources/static/plone.less
+  <https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone.less>`_
+- `plone/staticresources/static/plone-logged-in.js
+  <https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone-logged-in.js>`_
+- `plone/staticresources/static/plone-logged-in.less
+  <https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone-logged-in.less>`_
+
+When editing a pattern (e.g. in ``mockup`` package), it's important to know for which bundle it's registered, which can
+be done by checking where it's imported in the entry points above. The bundle name will be useful as parameter for
+``plone-compile-resources`` script.
+
+.. note::
+  Patterns are also available as independent resources in ``Resource Registries`` control panel but they are not
+  registered to default Plone bundles, which instead use the entry point approach. Pattern's resources may be useful if
+  needed to be injected in specific views.
+
+In production mode (``./bin/instance start``), since version 5.1, Plone is configured to serve Aggregate Bundles,
+single files to minimize network requests (`aggregate bundles
+<https://docs.plone.org/adapt-and-extend/theming/resourceregistry.html#resource-bundle-aggregation>`_).
+In that case final production resource URLs will be similar to:
+
+- http://localhost:8080//++plone++production/++unique++TIMESTAMP/default.js
+- http://localhost:8080//++plone++production/++unique++TIMESTAMP/default.css
+
+If you enable Development Mode for JavaScript and CSS in Resource Registries control panel, Plone will omit timestamp
```

Comment:
Perhaps use the [`:guilabel:` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel) for "Development Mode" and "Resource Registries"?